src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
```

Comment:
I recommend moving the interfaces to another file. Additionally, consider splitting them into separate files, such as `user.model.ts`, `post.model.ts`, and `comment.model.ts`, for better organization and maintainability.